modules/update-upgrading-cli.adoc

```diff
@@ -116,75 +115,36 @@ $ oc adm upgrade --to=<version> <1>
 +
 [source,terminal]
 ----
-$ oc get clusterversion -o json|jq ".items[0].spec"
+$ oc adm upgrade
 ----
 +
 .Example output
 [source,terminal]
 ----
-{
-  "channel": "stable-4.10",
-  "clusterID": "990f7ab8-109b-4c95-8480-2bd1deec55ff",
-  "desiredUpdate": {
-    "force": false,
-    "image": "quay.io/openshift-release-dev/ocp-release@sha256:9c5f0df8b192a0d7b46cd5f6a4da2289c155fd5302dec7954f8f06c878160b8b",
-    "version": "4.10.13" <1>
-  }
-}
-----
-<1> If the `version` number in the `desiredUpdate` stanza matches the value that
-you specified, the update is in progress.
+$ oc adm upgrade
```

Comment:
```suggestion

```

Reply:
Thanks; fixed in 9afeaa720e -> 37cc657405.